packages/next/cli/next-start.ts

```diff
@@ -43,18 +45,22 @@ const nextStart: cliCommand = (argv) => {
       Options
         --port, -p      A port number on which to start the application
         --hostname, -H  Hostname on which to start the application
+        --socket, -S    Unix socket path to listen on
         --help, -h      Displays this message
     `)
     process.exit(0)
   }
 
   const dir = resolve(args._[0] || '.')
   const port = args['--port'] || 3000
-  startServer({ dir }, port, args['--hostname'])
+  const hostname = args['--hostname']
+  const socket = args['--socket']
+
+  startServer({ dir }, ...(socket ? [socket] : [port, hostname]))
     .then(async (app) => {
-      Log.ready(
-        `started server on http://${args['--hostname'] || 'localhost'}:${port}`
-      )
+      if (!socket) {
```

Comment:
Why not log if socket is used? I think it should be:

```js
Log.ready(`started server on ${socket || `http://${args['--hostname'] || 'localhost'}:${port}`}`)
```

Reply:
I rebased and made this change for both next-dev and next-start.